### ISSUE TITLE
Release update for doodleverse_utils version 0.0.17 with bug fixes

### ISSUE THREAD

**Issue by 2320sharon** [Doodleverse/doodleverse_utils#20]
Recently a PR that fixed issue #18  was merged thanks to @ebgoldstein . Can you update the pypi package be updated to reflect the new bug fixes?

I think I found the cause of the `float_16` resize bug in #18 it was caused by activating Keras's mixed precision in downstream applications which uses `float16` instead of `float32`. If mixed precision was activated in a downstream application like `coastseg` or `segmentation_zoo` it would change the datatype of the estimated label `est_label` to float16 instead of the expected float32.
"Today, most models use the float32 dtype, which takes 32 bits of memory. However, there are two lower-precision dtypes, float16 and bfloat16, each which take 16 bits of memory instead"-[Keras Documentation on Mixed Precision](https://keras.io/api/mixed_precision/)

**Comment by 2320sharon** [comment]
@ebgoldstein  Thank you for updating the package!

**Comment by ebgoldstein** [comment]
@2320sharon , we are now on v0.0.18.
![image](https://user-images.githubusercontent.com/5330599/206456912-b337d57b-c1eb-4e7e-ae4f-8c13613fe81c.png)

i will let you close the issue if it works for you. Thanks for all the help here!

**Comment by 2320sharon** [comment]
I've tested it and it looks like it's working! Thank you again @ebgoldstein
